pkg/kubelet/cm/topologymanager/policy.go

```diff
@@ -147,19 +174,18 @@ func compareHints(bestNonPreferredAffinityCount int, current *TopologyHint, cand
 
 	// If the current bestHint is preferred and the candidate hint is
 	// non-preferred, never update the bestHint, regardless of the
-	// candidate hint's narowness.
+	// candidate hint's fitness.
```

Comment:
I'm still not sold on the word fitness here. Need to come up with something better.

Reply:
I agree. I spent some time trying to figure out more appropriate word but I was unsuccessful 😞

Reply:
I think we can just say:
```
// If the current bestHint is preferred and the candidate hint is
// non-preferred, never update the bestHint, regardless of how
// the candidate hint's affinity mask compares to the current
// hint's affinity mask.
```